docs/CONFIGURING-SLACK.md

```diff
@@ -16,14 +16,9 @@
  - channels:join
  - channels:manage
  - channels:read
- - channels:history
```

Comment:
This key is not mentioned in the doc file.
is this exclusion correct? 
![image](https://user-images.githubusercontent.com/44374637/89187144-3f68a780-d573-11ea-8a4c-d3af1aff9e4c.png)

Reply:
This scope was not used in any environment, it was only on the documentation.